test/generators/java/JavaGenerator.spec.ts

```diff
@@ -75,27 +80,37 @@ describe('JavaGenerator', () => {
     const expected = `public class CustomClass {
   @JsonProperty("property")
   private String property;
+  @JsonProperty("additionalProperties")
+  private Map<String, Object> additionalProperties;
 
   @JsonProperty("property")
   public String getProperty() { return this.property; }
   @JsonProperty("property")
   public void setProperty(String property) { this.property = property; }
+
+  @JsonProperty("additionalProperties")
+  public Map<String, Object> getAdditionalProperties() { return this.additionalProperties; }
+  @JsonProperty("additionalProperties")
+  public void setAdditionalProperties(Map<String, Object> additionalProperties) { this.additionalProperties = additionalProperties; }
 }`;
 
     generator = new JavaGenerator({ presets: [
       {
         class: {
           property({ renderer, propertyName, content }) {
             const annotation = renderer.renderAnnotation('JsonProperty', `"${propertyName}"`);
-            return `${annotation}\n${content}`;
+            return `${annotation}
+${content}`;
           },
           getter({ renderer, propertyName, content }) {
             const annotation = renderer.renderAnnotation('JsonProperty', `"${propertyName}"`);
-            return `${annotation}\n${content}`;
+            return `${annotation}
+${content}`;
           },
           setter({ renderer, propertyName, content }) {
             const annotation = renderer.renderAnnotation('JsonProperty', `"${propertyName}"`);
-            return `${annotation}\n${content}`;
+            return `${annotation}
+${content}`;
```

Comment:
I don't like this approach, because then you must write the next line from beginning of line, we should stay with `\n`.

Reply:
Sorry! This was an unintended side effect of a search/replace 😄 Gonna revert this